apps/documents/rpc.py

```diff
@@ -46,3 +46,24 @@ def add_document_from_s3(self, bucket, key, vault_wallet, storage_credentials, v
         log_metric('transmission.error', tags={'method': 'event_rpcclient.subscribe', 'code': 'RPCError',
```

Comment:
```suggestion
        log_metric('transmission.error', tags={'method': 'document_rpcclient.add_document_from_s3', 'code': 'RPCError',
```
Please fix any other metrics that are wrong like this